radix-engine-interface/src/model/package/access_rules.rs

```diff
@@ -0,0 +1,31 @@
+use crate::api::types::{AccessRuleKey, MetadataMethod, NativeFn, NativeMethod, PackageMethod};
+use crate::scrypto;
+use sbor::*;
+
+#[derive(Debug, PartialOrd, Ord, PartialEq, Eq, Clone)]
+#[scrypto(TypeId, Encode, Decode)]
+pub enum PackageMethodAuthKey {
```

Comment:
Though this does make this interface temporarily cleaner I think this will need to get refactored when we implement "Associated Nodes". So can we revert this for now?

Reply:
Sounds good, will revert this change.